internal/providers/pke/pkeworkflow/create_aws_worker_pool_activity.go

```diff
@@ -94,6 +94,10 @@ func (a *CreateWorkerPoolActivity) Execute(ctx context.Context, input CreateWork
 		return "", emperror.Wrap(err, "loading CF template")
 	}
 
+	if input.Pool.SpotPrice == "0.0" {
```

Comment:
if the zero value in `input.Pool.SpotPrice` can take various forms (e.g. `0.0; 0; .0`)  than it's safer to use something like:
```
if p, err := strconv.ParseFloat(input.Pool.SpotPrice, 64); err != nil || p == 0.0 {
  input.Pool.SpotPrice = ""
}
```

Reply:
Already done 😄